index.md

```diff
@@ -14,11 +14,83 @@ toc_sticky: true
 sidebar:
   - title: "Fetal CMR Resources"
     text: "a list of technical resources for fetal cardiac magnetic resonance imaging"
-last_modified_at: 2021-04-08T00:00:00-04:00
+last_modified_at: 2021-04-23T22:00:00-04:00
 ---
 
-**Notice:** This page is intended as a repository of technical resources put together by the [SCMR Fetal CMR and CMR in Pregnancy Special Interest Group](https://scmr.org/members/group.aspx?id=210568). Content will be added following the first Fetal CMR + CMR in Pregnancy SIG Virtual Session on April 9, 2021.
```

Comment:
It would probably be good to leave some type of notice on the front page. Maybe just delete the second line: 'Content will be added...'

Reply:
Good point. I'll remove the notice here and add the information on the right sidebar instead.